pages/spicedb/concepts/schema.mdx

```diff
@@ -38,8 +38,9 @@ definition someobjecttype {}
 ```
 
 <Callout type="info">
-  Note that the examples are unprefixed.
-  You'll need to add the prefix from your permissions system if calling `WriteSchema` for a permissions system hosted in Serverless.
+  Note that the examples are unprefixed. You'll need to add the prefix from your
```

Comment:
one sentence per line

Reply:
Can we fix this in a followup when we add the linter?